Doc/howto/regex.rst

```diff
@@ -1284,12 +1292,12 @@ doesn't work because of the greedy nature of ``.*``. ::
    >>> print(re.match('<.*>', s).group())
    <html><head><title>Title</title>
 
-The RE matches the ``'<'`` in ``<html>``, and the ``.*`` consumes the rest of
+The RE matches the ``<`` in ``'<html>'``, and the ``.*`` consumes the rest of
```

Comment:
I think you do want ``'<'`` as in the original here. See also line 1299 in this commit.